### ISSUE TITLE
Debugging paper size errors

### ISSUE THREAD

**Issue by triblondon** [williamkapke/ipp#20]
Thanks for this module. It seems to be exactly what I'm looking for but try as I might my stubborn Brother QL-570 refuses to emit a single label.  I have tried, I really have.  So far I have tried:

* Creating a PDF with the exact paper dimensions advertised by the printer (or as close as I can get in postscript points)
* Both orientations, both on the PDF and on the IPP `orientation-requested` attribute
* Specifying (and not specifying) a `media` attribute set to the appropriate value obtained from a `Get-Printer-Attributes` call
* Turning it off and on again
* Singing to it

Nothing's worked so far.  Here's my code:

```javascript
var ipp = require('ipp');
var PDFDocument = require('pdfkit');
var bufs = [];

var doc = new PDFDocument({
    size: [175.691, 283.436], // 100 x 62 mm in 'PDF points'
    margin:0,
    layout: 'landscape'
});

doc.text("Hello world", 0, 0);
doc.on('data', function(chunk) { bufs.push(chunk); });
doc.on('end', function() {
	var pdf = Buffer.concat(bufs);
	var printer = ipp.Printer("ipp://localhost/printers/Brother_QL_570");
	var msg = {
		"operation-attributes-tag": {
			"requesting-user-name": "Andrew",
			"job-name": "Test",
			"document-format": "application/pdf"
		},
		"job-attributes-tag":{
			"copies": 1,
			"sides": "one-sided",
			"media": "om_62mm_61.98x99.99mm",
			"orientation-requested": "landscape"
		},
		"data": pdf
	};
	printer.execute("Print-Job", msg, function(err, res){
		console.log(res);
	});
});
doc.end();
```

I get a success response, but when I run a `Get-Job-Attributes` on it, the output includes:

```
     'job-state': 'processing',
     'job-state-reasons': 'job-printing',
     'job-media-sheets-completed': 32767,
     'job-k-octets': 1,
     'job-hold-until': 'no-hold',
     'job-sheets': [ 'none', 'none' ],
     'job-printer-state-message': 'Sending data to printer.',
     'job-printer-state-reasons': 'com.brother-paper-size-error'
```

My use case is printing name badges for a tech meetup - I'd like them to be printed automatically when people are checked in using Eventbrite.  I have the whole system working except this bit. I'm sur eyou have better things to do but if you have any advice I'd really appreciate it.

**Comment by triblondon** [comment]
OK, I figured it out.  The problem was that I had the wrong media name.  The name I needed wasn't reported by the printer in the `Get-Printer-Attributes` call, so I ended up finding it like this:

1. printing from Preview using the OSX print dialog
2. finding the print job number in the CUPS web UI
3. viewing the spool file in /var/spool/cups (as root), which is a hard to read file with a lot of unprintable characters, but most of the actual data is ASCII.
4. finding the 'media' property, which was set to 'DC07'

So I set the 'media' property of the msg object to `DC07` and now it works!
